_typos.toml

```diff
@@ -0,0 +1,2 @@
+[default.extend-words]
+cll = "cll"
```

Comment:
I'm not sure what benefit this file will bring us. Please remove it from th PR.

Reply:
It's avoid Typos tools (locally enable in editor and/or in ci) to report errors like this :

> error: `cll` should be `call`
>   --> ./src/lib.rs:775:18
>     |
> 775 |         self.max_cll
>     |                  ^^^
>     |
> error: `Cll` should be `Call`
>   --> ./src/element_id.rs:261:32
>     |
> 261 |         m.insert(ElementId::MaxCll, ElementType::Unsigned);

Reply:
But, why do you assume that everybody uses this tool? Can't you override it locally? It's also not part of the CI as far as I can tell. Please reduce the PR to just fix the typos you found.

Reply:
I updated the PR.

I don't assume that everybody use this tool, it's more a _proposal_ to use it.